### PR TITLE
Diana2

### PR DESCRIPTION
**Pull request by dluangrath**
Added ellipsis options menu in Task to display delete, duplicate menu items

### CODE REVIEW

File: Client/src/App.jsx [hunk 1]
```diff
@@ -27,12 +27,12 @@ export default function App() {
     <Router>
       {/* <UserContext.Provider value={{ userData, userLoading, userError }}> */}
       <Routes>
-        <Route path = "/" element = {<SignUp />} />
-        <Route path = "/Login" element = {<Login />} />
-
-        <Route element={ <PrivateRoutes/>}>
+        {/* <Route path = "/" element = {<SignUp />} />
```
Comment (Matthew-Allen-1) [inline Client/src/App.jsx:30]: Please remove the commenting out of the other routes here.  That's a good way to get around having to sign in during development, but we want the sign in page to be the first page loaded generally.

File: Client/src/Components/GroupedTask/GroupedTask.jsx [hunk 2]
```diff
@@ -127,25 +130,6 @@ export default function GroupedTask(props){
     )}
   })
 
-  // console.log('taskElementArrays', taskElementArrays)
```
Comment (Matthew-Allen-1) [inline Client/src/Components/GroupedTask/GroupedTask.jsx:130]: It looks like this section didn't make it through the merge conflict when you merged the master branch into your branch, but it is necessary for calculating the elapsed times of each task.  Could you copy and paste the code from here into your GroupedTask.jsx file then make a new commit and pull request?

File: Client/src/Components/GroupedTask/GroupedTask.jsx [hunk 3]
```diff
@@ -160,8 +144,8 @@ export default function GroupedTask(props){
           {selectAll && <input type="checkbox"/>}
           <p className = "left" >{dateStr}</p>
           <div className = "right">
-            <p>Total Time: {convertElapsedToText(timeTotals[index])}</p>
```
Comment (Matthew-Allen-1) [inline Client/src/Components/GroupedTask/GroupedTask.jsx:163]: This change also didn't seem to make it through your merge.  Could you do the same here, copy and paste the deleted version in red and replace the unrevised version in your file before making a new commit and pull request?

File: Client/src/Components/Task/Task.jsx [hunk 4]
```diff
@@ -116,7 +123,7 @@ export default function Task(props){
           />
           <span className="line-divider"></span>
           <span className="elapsed-time">Time: {props.elapsedTime}</span>
-          <img className="options" src="https://app.clockify.me/assets/ui-icons/menu-dots-vertical.svg" alt="" />
```
Comment (Matthew-Allen-1) [inline Client/src/Components/Task/Task.jsx:119]: It seems that the three dots image was deleted here?  Was that purposeful?

**Review by Matthew-Allen-1 — CHANGES_REQUESTED** [review]
Hi, Diana.  I just looked over your pull request and found a few minor issues with code that had been changed but didn't seem to make it into your branch when you merged in the master branch yesterday.  I attached comments to a few places in your pull request.  Could you make these adjustments then make a new commit and pull request?  I hope my comments are clear and readable to you.  This my first time making comments on a pull request, so I don't really know what I'm doing, but I hope I'm doing this right!

File: Client/src/Components/GroupedTask/GroupedTask.jsx (hunk 3 above)
Comment (dluangrath) [reply Client/src/Components/GroupedTask/GroupedTask.jsx:163]: copied & pasted deleted version in red

File: Client/src/Components/Task/Task.jsx (hunk 4 above)
Comment (dluangrath) [reply Client/src/Components/Task/Task.jsx:119]: Yes I replaced the image with OptionsMenu component

**Review by Matthew-Allen-1 — APPROVED** [review]

**Pull request merged**